symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py

```diff
@@ -3,8 +3,8 @@
                            validate_output, dimensionless)
 
 # Description
-## The optical path length between two points of the medium is the distance to which light (optical radiation) would propagate in a vacuum during its passage between these points.
-## Since the speed of light in any medium is less than its speed in a vacuum, the optical path length is always greater than the actual distance traveled.
+## The optical path length L between points A and B of a transparent medium is the distance over which light (optical radiation) has spread it would be in a vacuum for the same time it takes from A to B in the medium.

```

Comment:
If L is between A an B and it is not curved, it is the same path as line between A and B in medium.